.github/workflows/release.yaml

```diff
@@ -27,14 +29,21 @@ jobs:
         uses: imranismail/setup-kustomize@v1
         with:
           kustomize-version: "3.5.4"
-      - name: manifest-debug
-        run: make manifest
-      - name: find-debug
-        run: find ./out
+      - name: Set up Docker Buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
```

Comment:
I am a little concerned about putting an action from some unknown person in our supply chain. Is there a more standardized way to get it? A `docker/*` action or GitHub `actions/*` action?

Reply:
`crazy-max` is the same person that maintains the `goreleaser` one, but I understand your concerns. I took it from the GitHub marketplace, I will look for something different if exists

Reply:
It is worth a 5-10 min search to see if anything from a corporate source is available. If not, we can live with it. I expect eventually the docker actions will have it built-in